pr-template.md

```diff
@@ -0,0 +1,13 @@
+#### What's this PR do?
+
+#### Where should the reviewer start?
+
+#### How should this be manually tested?
+
+#### Any background context you want to provide?
+
+#### What are the relevant tickets?
```

Comment:
I'd like to move this up the list, maybe even to the first position. 

We should also use the github keywords for [closing issues via commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/)